### PR TITLE
feat!: add new variables

### PR DESCRIPTION
**Pull request by waxb**
- chore: changelog updated
- feat!: add new variables

### CODE REVIEW

File: README.tfdoc.hcl [hunk 1]
```diff
@@ -133,6 +133,22 @@ section {
             Maximum Transmission Unit in bytes. The minimum value for this field is `1460` and the maximum value is `1500` bytes.
           END
         }
+
+        variable "enable_ula_internal_ipv6" {
+          type        = bool
+          default     = false
+          description = <<-END
+            Enable ULA internal ipv6 on this network. Enabling this feature will assign a /48 from google defined ULA prefix fd20::/20.
```
Comment (soerenmartius) [inline README.tfdoc.hcl:141]:
```suggestion
            Enable ULA internal ipv6 on this network. Enabling this feature will assign a `/48` from Google defined ULA prefix `fd20::/20`.
```

File: README.tfdoc.hcl [hunk 2]
```diff
@@ -133,6 +133,22 @@ section {
...
+          END
+        }
+
+        variable "internal_ipv6_range" {
+          type        = string
+          default     = null
+          description = <<-END
+            When enabling ula internal ipv6, caller optionally can specify the /48 range they want from the google defined ULA prefix fd20::/20. The input must be a valid /48 ULA IPv6 address and must be within the fd20::/20. Operation will fail if the speficied /48 is already in used by another resource. If the field is not speficied, then a /48 range will be randomly allocated from fd20::/20 and returned via this field."
```
Comment (soerenmartius) [inline README.tfdoc.hcl:149]:
```suggestion
            When enabling ula internal ipv6, caller optionally can specify the `/48` range they want from the Google defined ULA prefix `fd20::/20`. The input must be a valid `/48` ULA IPv6 address and within the `fd20::/20`. The operation will fail if the specified `/48` is already in use by another resource. If the field is not specified, then a `/48` range will be randomly allocated from `fd20::/20` and returned via this field."
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Corrected some typos (probably copy & paste from the already messed up Google docs) and added formatting for the README.

**Review by soerenmartius — APPROVED** [review]
lgtm! thanks. 

Maybe consider adding the new values to the complete test

**Comment by waxb** [comment]
> lgtm! thanks.
> 
> Maybe consider adding the new values to the complete test

they were already

Screenshot 2022-11-05 at 16 37 55

thank you for the review!

**Pull request merged**